libsawtooth/Cargo.toml

```diff
@@ -61,6 +64,7 @@ experimental = [
 ]
 
 btree-store = ["stores"]
+client = ["reqwest", "serde"]
```

Comment:
This should probably be split into two features:

```toml
client = [] # covers the trait, 
client-rest = ["client", "reqwest", "serde"]  # or `client-http`, to cover the REST/HTTP client implementation
```

Reply:
fixed